packages/components/src/search-control/index.tsx

```diff
@@ -28,8 +30,8 @@ function SearchControl(
 		hideLabelFromVision = true,
 		help,
 		onClose,
-	},
-	forwardedRef
+	}: SearchControlProps,
+	forwardedRef: ForwardedRef< any >
 ) {
 	const searchRef = useRef();
```

Comment:
This should fix a couple of issues for you:

1. You would not need to use `if ( searchRef.current )`
2. No casting is needed in `searchRef.current as HTMLInputElement`

```suggestion
	const searchRef = useRef< HTMLInputElement >();
```

Reply:
Great suggestion! Committed your suggestion.